lib/debug.js

```diff
@@ -1,11 +1,11 @@
-var log = require('debug')
-var debug
-
-function init () {
-  if (!process.env.DEBUG) {
-    log.enable('z2m:*')
-  }
-  debug = log('z2m')
+var debug = require('debug')
+
+if (!process.env.DEBUG) {
+  debug.enable('z2m:*')
 }
-init()
+
+debug = debug('z2m')
+
+debug.log = console.log.bind(console)
```

Comment:
Without this debug outputs to stderr @chrisns . You should add back this line. When using tools like pm2 everything will go to error log file.